src/json/state_machine_input.json

```diff
@@ -119,6 +119,27 @@
       ]
     }
   },
+  "vpcpreconfig":{  
+    "credentials":null,
+    "federation":{  
+       "roles":[  
+
+       ],
+       "authorizer_user_guid":null,
+       "durationSeconds":0
+    },
+    "manageVpcPreConfig":{  
+       "account":null,
+       "region":null,
+       "regions":[  
```

Comment:
If tomorrow we have one more region will that automatically get addressed in the ManagedVPC preconfig statemachine

Reply:
Yes sonal it will be addressed.